README.rst

```diff
@@ -33,3 +33,14 @@ Bug tracker
 ===========
 A bug tracker is available at
 https://github.com/dataflake/dataflake.fakeldap/issues
+
+Building and Deploying
+======================
+
+Requirements:
+  - A Packagecloud.io key with Environment variable set to it, such as PACKAGE_CLOUD_KEY="PACKAGECLOUDKEY"
```

Comment:
why the caps?

Reply:
and that might as well be a link. i'd consider adding a link to our wiki as well but leaning against it as this fork could be made public.